packages/devtools_app/lib/src/screens/inspector_v2/widget_properties/properties_view.dart

```diff
@@ -96,21 +97,44 @@ class PropertiesTable extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final sortedProperties = _sortPropertiesByLevel(properties);
```

Comment:
if there is a chance that this operation will be expensive (for example if N is large for N properties), then we should make this a stateful widget and only do this sorting in initState and didUpdateWidget

Reply:
I don't think there is ever more than ~30 properties, but moved to `initState` and `didUpdateWidget` to be safe